output/xml/default.xml

```diff
@@ -179,6 +179,7 @@
 		<event name="OnPaint" class="wxPaintEvent" help="Process a wxEVT_PAINT event." />
 		<event name="OnSize" class="wxSizeEvent" help="Process a wxEVT_SIZE event." />
 		<event name="OnUpdateUI" class="wxUpdateUIEvent" help="Process a wxEVT_UPDATE_UI event." />
+		<event name="OnMove" class="wxMoveEvent" help="Process a wxEVT_MOVE event, which is generated when a window is moved. Not generated in wxGTK" />
```

Comment:
wxFB displays the events in the order they appear in the xml file, OnSize and OnMove are kind of related and now separated by OnUpdateUI. Can you place it between OnSize and OnUpdateUI instead please and also update the order in the code templates accordingly?

Reply:
Fixed